vars/util.groovy

```diff
@@ -204,4 +204,23 @@ GIT INFORMATION REPORT
     } else {
         println '[WARNING] The variable GIT_INFORMATION_REPORT does not exist'
     }
+}
+
+def getNextVersion(String version, String type, String suffix = 'SNAPSHOT') {
+    assert ['major', 'minor', 'micro'].contains(type)
```

Comment:
Can you add a test and checking the assert is returning an error if type is not one of the item of the list ?